docs/source/tutorials/linops/linear_operators_quickstart.ipynb

```diff
@@ -0,0 +1,622 @@
+{
```

Comment:
<p>are built from their ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/534/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I think we need a relative clause here, don't we? </p>

Reply:
<p>Correct, but the sentence is in passive, which is why the past tense is used, i.e. "built". See also https://www.linguee.com/english-german/translation/houses+are+built.html</p>

Reply:
ah, I thought you wanted me to skip the word "that" :bulb: . corrected.